nm_openconnect.py

```diff
@@ -114,7 +115,7 @@ def Connect(self, connection: dict[str, dict[str, Any]]):
         cmd = [
             'openconnect', '-u', self.username, '--passwd-on-stdin',
             '--script', '/usr/lib/nm-openconnect-service-openconnect-helper',
-            '--syslog', '--protocol', 'anyconnect', *self.form_data,
+            '--syslog', '--protocol', self.protocol, *self.form_data,
             self.gateway
```

Comment:
The issue is thare is a the situation when `protocol` has not been set in `*.nmconnection` file. There are two options.

+ The first one is easy. We set `self.protocol` with `anyconnect` by default on initialization or on `NeedSecret` event (or `SetConfig` event?).
+ The second option is to add `--protocol` option if it is non-trivial (i.e. `None` or `'anyconnect'`). This behaviour is closer to original `nm-openconnect` pluging (see [these lines][1]).

[1]: https://gitlab.gnome.org/GNOME/NetworkManager-openconnect/-/blob/1.2.10/src/nm-openconnect-service.c?ref_type=tags#L496-504